fpga: Updated readme (#497)
pulp-platform/snitch: hw/system/occamy/fpga/README.md

# HerOccamy

## Generating the Bitstream

To create the bitstream for Occamy on the VCU128, you currently need to follow three steps.

### Start by compiling HERO

The hero repository contains the GCC toolchain (to build the kernel) and the LLVM toolchain (for applications).

Your need to compile this at first as the bootrom will need to store `uboot-spl` given by hero.

> And the flash will need to store `uboot` which is given by hero too.

Reviewer: ```suggestion
Furthermore, the flash will require `uboot` from hero.
```

Author: Thanks ! ^^
